src/sa_web/static/js/views/map-view.js

```diff
@@ -40,12 +43,25 @@ var Shareabouts = Shareabouts || {};
         } else if (config.type && config.type === 'landmark') {
           collectionId = config.id;
           self.layers[collectionId] = L.layerGroup();
-          self.landmarkLayerViews[collectionId] = {};
+
+          if (config.visibleDefault) {
+            self.map.addLayer(self.layers[collectionId]);
+          }
+          // REFACTOR
+          // add landmark layer views to the layerViews object:
+          self.layerViews[collectionId] = {};
 
           // Bind our landmark data events:
-          collection = self.options.landmarkCollections[collectionId];
-          collection.on('add', self.addLandmarkLayerView(collectionId), self);
-          collection.on('remove', self.removeLandmarkLayerView(collectionId), self);
+          //collection = self.options.landmarkCollections[collectionId];
+          
+          // filter out places collection from meta-collection object
```

Comment:
This may have been partly due to sloppy coding on my part, because the `var collection` should probably be defined only within this `config.type && config.type === 'landmark'` conditional block, and not above. We don't really use the collection, other than to bind the 'add' and 'remove' events.